TAT/Models/Course.swift

```diff
@@ -40,10 +43,13 @@ class Course: NSObject, Codable {
       #endif
     }
   }
+
```

Comment:
Should I leave a blank line above last right brace and down before everything in a class, or just personal preference, like these:
``` swift
class Foo {

  // MARK: - Bar

}
```
and 
``` swift
func foo() {
  while bar {
    // some codes here
  }

}
```

Reply:
yes you should do this, also leave a blank line in the vary end of class, extension, and struct.